### PR TITLE
tkt-56466: Netcli runs on virtual terminal 0 only

### PR DESCRIPTION
**Pull request by sonicaj**
This commit ensures that netcli runs on virtual terminal 0 only and other virtual terminals remain unaffected by it.
Ticket: #56466

### CODE REVIEW

File: src/freenas/etc/ix.rc.d/ix-ttys [hunk 1]
```diff
@@ -40,13 +40,16 @@ update_ttys()
 			serspeed=""
 		fi
 
+		# Let's make sure that all virtual terminals are set to Pc before we make any changes
+		sed -E -e "s,^(ttyv.*)(freenas|freenas.115200|Pc)(.*)\$,\1Pc\3," /etc/ttys > /etc/ttys.bak && mv /etc/ttys.bak /etc/ttys
```
Comment (william-gr) [inline src/freenas/etc/ix.rc.d/ix-ttys]: Why mv? Wouldnt sed -i '' have the same effect?

Comment (sonicaj) [reply src/freenas/etc/ix.rc.d/ix-ttys]: It will create a backup file first

Comment (sonicaj) [reply src/freenas/etc/ix.rc.d/ix-ttys]: Both approaches will work. We will need to delete the backup file then if we use -i. What do you think would be the best approach ? I can update accordingly

Comment (william-gr) [reply src/freenas/etc/ix.rc.d/ix-ttys]: I wont, thats why empty -i, see `man sed`

**Pull request merged**